src/tools/generate-copyright/src/cargo_metadata.rs

```diff
@@ -52,14 +52,13 @@ pub struct PackageMetadata {
 /// assume `reuse` has covered it already.
 pub fn get_metadata_and_notices(
     cargo: &Path,
-    dest: &Path,
+    vendor_path: &Path,
```

Comment:
The doc comment above this function is not a bit imprecise, as it mentions `${BUILD}/vendor`.